web/src/utils/refetching.ts

```diff
@@ -39,7 +40,6 @@ export function getStaleTime(timeRange: TimeRange, urlDatetime?: string) {
     return 0;
   }
   const now = new Date();
-  const nextHour = new Date(now);
-  nextHour.setHours(nextHour.getHours() + 1, 0, 0, 0);
```

Comment:
This used to be flooring to the start of the hour, but the new version is not. As far as I remember, we do want it to be start of hour specifically for the refetching, right? :)

Reply:
Not sure but since we are using start of hour elsewhere and to maintain the same functionality I'll just add that as well.